pilot/pkg/security/authz/builder/extauthz.go

```diff
@@ -94,16 +91,17 @@ func processExtensionProvider(in *plugin.InputParams) (map[string]*builtExtAuthz
 		if err != nil {
 			errs = multierror.Append(errs, multierror.Prefix(err, fmt.Sprintf("failed to parse extension provider %q:", config.Name)))
 		}
+		if parsed == nil {
```

Comment:
couldn't you init parsed as &buildExtAuthz in line 80?

Reply:
the buildExtAuthzHTTP/buildExtAuthzGRPC might return nil, I didn't want to change those 2 functions as it seems nicer to keep the convention of returning nil on errors in that scope.